README.md

```
@@ -1,13 +1,177 @@
-# React PageFlipper
+[![GitHub license](https://img.shields.io/github/license/samehben3li/react-pageflipper)](https://github.com/samehben3li/react-pageflipper/blob/master/LICENSE) [![npm](https://img.shields.io/npm/v/react-pageflipper)](https://www.npmjs.com/package/react-pageflipper) [![npm](https://img.shields.io/npm/dm/react-pageflipper)](https://npmcharts.com/compare/react-pageflipper?minimal=true)
```

markdownlint-cli2 (0.18.1)

First line in a file should be a top-level heading

(MD041, first-line-heading, first-line-h1)